src/main/java/io/cdap/plugin/gcp/gcs/actions/SourceDestConfig.java

```diff
@@ -125,9 +136,20 @@ public void validate(FailureCollector collector, Map<String, String> arguments)
     if (!containsMacro(NAME_CMEK_KEY)) {
       validateCmekKey(collector, arguments);
     }
+    if (!containsMacro(READ_TIMEOUT)) {
+      validateReadTimeout(collector);
+    }
     collector.getOrThrowException();
   }
-
```

Comment:
why is this removed? There should be an empty line between two methods.

Reply:
Done